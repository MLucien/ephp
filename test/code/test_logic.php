2 ^ 3 = <?=2^3?>
1 & 1 = <?=1&1?>
2 | 1 = <?=2|1?>

$a = 5; <? $a = 5; ?>

$a ^ 1 = <?=$a^1?>
$a & 1 = <?=$a&1?>
$a | 1 = <?=$a|1?>

