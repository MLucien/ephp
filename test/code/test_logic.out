2 ^ 3 = 11 & 1 = 12 | 1 = 3
$a = 5; 
$a ^ 1 = 4$a & 1 = 1$a | 1 = 5
